graylog2-web-interface/src/logic/rest/FetchProvider.ts

```diff
@@ -76,6 +76,16 @@ const defaultResponseHandler = (resp: Response) => {
   throw resp;
 };
 
+const streamingPlainTextResponseHandler = async (resp: Response) => {
```

Comment:
Yes, we can remove the async. I think we can keep the handler and rename it `textResponseHandler` to avoid some logic to determine whether it's a text response or not.